doc/palette.md

```diff
@@ -54,7 +72,10 @@ class Palette {
 }
 ```
 
-A `PaletteEntry` is a const class that can hold information of a color, and has two attributes:
+A `PaletteEntry` is a const class that holds information of a color and it has the following
+members:
 
- * `color`: returns the `Color` specified
- * `paint`: creates a new `Paint` with the color specified. `Paint` is a non-const class, so this attribute actually creates a brand new instance every time it's called. It's safe to cascade mutations to this.
+ - `color`: returns the `Color` specified
+ - `paint`: creates a new `Paint` with the color specified. `Paint` is a non-const class, so this
```

Comment:
```suggestion
 - `paint`: creates a new `Paint` with the color specified. `Paint` is a non-constant class, so this
```

Reply:
I don't think it makes it clearer to spell it out, since the keyword in dart is `const`, but maybe we should backtick it?